incubator/fluentd/values.yaml

```diff
@@ -31,7 +33,6 @@ papertrail:
   enabled: false
   host: logs3.papertrailapp.com
   port: 12785
-  cluster_name:
```

Comment:
Instead of removing the `papertrail.cluster_name` nested key entirely, shouldn't we break `cluster_name` out to be its own key as it is required?

Reply:
Added a commented out version so we still require it but it is present in the values.yaml